deps/GitSubmodules/GitSubmodules.vcxproj

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please absolutely don't do this.

You can just tell the checkout step to update submodules. It's much easier than adding a new C++ project that has.. a garbage file output and putting it in the solution and turning it into a workhorse that runs commands for you.